low-level/actuators/impl/generic/realstor_encl.py

```diff
@@ -599,7 +636,7 @@ def _get_encl_response(self, uri, request_type):
             return None
         if response.status_code != self.rssencl.ws.HTTP_OK:
             logger.error(f"Failed to get data for {uri}")
-            return None
+            return {}

```

Comment:
Made appropriate change.

Reply:
changes doesnt align with coding practices. 
Initializing var at the top, updating it in required cases and return can just use that var while returning, instead of feeding value at all places.

read my first comment to relate to code in here

Reply:
Done